WordPress/Docs/CodeAnalysis/EscapedNotTranslatedStandard.xml

```diff
@@ -0,0 +1,20 @@
+<documentation title="Escaped Not Translated Text">
+    <standard>
+    <![CDATA[
+    Text intended for translation needs to be wrapped in a localization function call.
+    This sniff will help you find instances where text is escaped for output, but no localization function is called, even though an (unexpected) text domain argument is passed to the escape function.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: esc_html__() used to translate and escape.">
+        <![CDATA[
+echo <em>esc_html__</em>( 'text', 'domain' );
```

Comment:
While this is a correct example, should we maybe add another correct example?

`esc_html_e( 'text', 'domain' );` ?

As this is just the shorthand for `echo esc_html__()` function.

https://developer.wordpress.org/reference/functions/esc_html_e/

Reply:
Sorry, no. That is not a function this sniff even looks at, so the example would be misleading.

Reply:
Ok, I guess I looked at it from a reviewer perspective - I always say to my colleagues to use appropriate functions instead 😄

Reply:
Oh, absolutely for reviewing "normal code".
There's actually a separate issue open to (at some point) create a sniff for that:#1315